app/src/main/java/ca/ubc/best/mint/museandroidapp/vm/StimulusCue.java

```diff
@@ -0,0 +1,242 @@
+package ca.ubc.best.mint.museandroidapp.vm;
+
+import android.util.Log;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Created by Carolfa on 2018-02-23.
+ */
+
+public class StimulusCue {
+    public FlankerStimulus stimulus;
+    public FlankerCue cue;
+    public StimulusCue (FlankerStimulus stimulus, FlankerCue cue) {
+        this.stimulus = stimulus;
+        this.cue = cue;
+    }
+
+    public static List<StimulusCue> createNeutralStimuliCueList(List<FlankerStimulus> neutralStimuliList) {
+        /** Neutral Stimuli are preceded by:
+            76.4% NULL cues
+            23.6% RP cues, out of all RP cues 84% are the correct cue, 16% are the wrong cue.
+            For the correct RP cues, 27.7% are followed by a catch stimuli "+++++", users are instructed to tap on the side indicated by the RP cue
+         **/
+        List<StimulusCue> neutralStimulusCueSet = new ArrayList<>();
+
+        int arraySize = neutralStimuliList.size();
+        int numNullCues = Math.round(0.764f * arraySize);
+        int numWrongRPCues = Math.round(0.236f * 0.16f * arraySize);
+        int numCorNCatRPCues = Math.round(0.236f * 0.84f * 0.723f * arraySize);
+        int numCorCatchRPCues =  Math.round(0.236f * 0.84f * 0.277f * arraySize);
+
+
+        /**Add null cues**/
+        int index = 0;
+
+        //half left neutral stimulus
+        while( index < numNullCues) {
+            neutralStimulusCueSet.add(new StimulusCue(neutralStimuliList.get(index), FlankerCue.NULL));
+            index ++;
+        }
+
+        //add wrong rp cues
+        while( index < numWrongRPCues) {
```

Comment:
Is this right? I feel like it should be:
for (int i = 0; i < numWrongRPCues; i++) {
 // etc...
 index++;
}

Say arraySize is 84:
numNullCues = Math.round(0.764 * 21) = Math.round(64.176) = 64
numWrongRPCues = Math.round(0.79296 * 84) = Math.round(3.17184) = 3

After the loop at lines 40-43, index will be 64 (= numNullCues)
This check will then fail immediately, because index < numWrongRPCues (i.e. 64 < 3) is false